settings.gradle

```diff
@@ -5,6 +5,9 @@ pluginManagement {
 		maven {
 			url 'https://repo.spring.io/plugins-release'
 		}
+		if (version.contains('-')) {
+			maven { url "https://repo.spring.io/milestone" }
+		}
```

Comment:
What's the motivation for adding this? We've thus far never had a milestone plugin dependency and I don't foresee having one in the future. Never say never, but I'd prefer to leave it out until it's needed.

Reply:
Ah, I see. It was meant for consistency, but I didn't realize that we never publish plugin milestones. I'll remove it.